terraform/01_provider.tf

```diff
@@ -41,3 +41,8 @@ terraform {
 
   }
 }
+
+provider "google" {
+  project = var.project_id
+  region  = var.appengine_region
```

Comment:
Hmm you're right that this could be a little confusing. The primary GKE cluster and load generator cluster explicitly set their own regions, which I assume would take precedence (but I should probably verify those assumptions).

Do you think we should remove the region from the provider, and just set it on each resource as needed?

Reply:
I think we can explicitly define this parameter for `provider` argument in the module call for rating service.
See [passing providers explicitly](https://www.terraform.io/language/modules/develop/providers#passing-providers-explicitly) example in the Terraform doc.

Reply:
good catch, I made the change